### PR TITLE
Fix 'Can't get tsserver to work #164'

### PR DESCRIPTION
**Pull request by igrep**
Fix https://github.com/neovim/nvim-lsp/issues/164

### CODE REVIEW

File: lua/nvim_lsp/tsserver.lua [hunk 1]
```diff
@@ -2,7 +2,12 @@ local configs = require 'nvim_lsp/configs'
 local util = require 'nvim_lsp/util'
 
 local server_name = "tsserver"
-local bin_name = "typescript-language-server"
+local bin_name
+if vim.fn.has('win32') then
+  bin_name = "typescript-language-server.cmd"
+else
+  bin_name = "typescript-language-server"
+end
```
Comment (h-michael) [inline lua/nvim_lsp/tsserver.lua:10]: You can write as follows:
```suggestion
local bin_name = "typescript-language-server"
if vim.fn.has('win32') then
  bin_name = bin_name..".cmd"
end
```

**Comment by igrep** [comment]
Thanks for a review! Fixed and rebased! ✅

**Comment by h-michael** [comment]
@igrep Thanks :)

**Comment by jhbabon** [comment]
Hi! I'm very sorry to step in this PR, but it seems that this fix broke any non windows integration. I created another PR to try to fix the issue in all platforms: https://github.com/neovim/nvim-lsp/pull/224

**Comment by igrep** [comment]
Oh... sorry! I've forgotten the behavior of Lua...

**Comment by neovim-discourse** [comment]
This pull request has been mentioned on **Neovim Discourse**. There might be relevant details there:

https://neovim.discourse.group/t/neovim-configs-and-windows/4734/3

**Pull request merged**